### PR TITLE
Add tests for JS 2 module

### PR DESCRIPTION
**Pull request by alismith-dev**
Adds questions for JS2 module.

### CODE REVIEW

File: module2.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/**
+  +++++++++++++++++++++++++++++++++++++++++++
+  Welcome to the JavaScript test (module 2)!
+  Please answer the following questions by filling in the parts marked with TODO:
+
+  The goal is to make all the tests pass!
+
+  Please UNCOMMENT each function, and its associated name in the module.exports
```
Comment (martynamaron) [inline module2.js:8]: Shall we uncomment them here as well?

File: module2.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+// function question4() {
+//   return {
+//     /* ??? */
+//   }
+// }
+
+/**
+  +++++++++++++++++++++++++++++++++++++++++++
+  Question 5:
+  Write a function that creates an click event listener on the `#btn` element.
```
Comment (martynamaron) [inline module2.js]: small typo :)

**Comment by martynamaron** [comment]
Fantastic job! You've added a whole new test so quickly, amazing! :)

File: module2.js (hunk 1 above)
Comment (alismith-dev) [reply module2.js:8]: Yep I'll do that

**Comment by alismith-dev** [comment]
@martynamaron Uncommented the implementation functions, fixed the typo and cleaned up the descriptions

**Pull request merged**